qiskit/extensions/qasm_simulator_cpp/rzz.py

```diff
@@ -17,7 +17,7 @@
 # =============================================================================
 
 """
-local_qiskit_simulator two-qubit ZZ-rotation gate.
+local_qasm_simulator two-qubit ZZ-rotation gate.
```

Comment:
This file should be moved to a standard qasm extension at some point.

Reply:
done.